docs/data/stashcache/run-stashcache-container.md

```diff
@@ -164,26 +157,26 @@ For caches that store over `10 TB` or that have assigned space for storing the c
              opensciencegrid/stash-cache:stable
 
 !!! note
-    Under this configuration the `<HOST PARTITION>` is not used to store the files rather to store symlinks to the files in `/partition1` and `/partition2`
+    Under this configuration the `<HOST PARTITION>` is not used to store the files.
+    Instead, the host partition stores symlinks to the files in `/partition1` and `/partition2`.
 
 !!! warning
     For over 100 TB of assigned space we highly encourage to use this setup and mount `<HOST PARTITION>` in solid state disks or NVME.
 
 
-Validating StashCache
+Validating the Cache
 ---------------------
 
-For example, if you've chosen `8212` as your host port, you can verify that it worked with the command:
+The cache server functions as a normal HTTP server and can interact with typical HTTP clients, such as `curl`.
+
 
 ```console
-user@host $ curl http://localhost:8212/user/dweitzel/public/blast/queries/query1
+user@host $ curl -O http://cache_host:8000/osgconnect/public/rynge/test.data
```

Comment:
Shouldn't the port be based off of `<HOST PORT>` from the above `docker run` command?

Reply:
Yeah that text kinda got lost. Let me fix...